shared/js/background/https.es6.js

```diff
@@ -85,8 +85,8 @@ class HTTPS {
             return reqUrl
         }
 
-        // Determine host
-        const host = utils.extractHostFromURL(reqUrl)
+        // Determine host without stripping 'www',
+        const host = utils.extractHostFromURL(reqUrl, true) || ''
 
         if (this.canUpgradeHost(host)) {
```

Comment:
Can you check that `host` exists here? Just in case line 89 sets the host to '' we get a bad bloom filter that decides '' is upgradable.

Reply:
@jdorweiler done, thanks